kickstarter/android-oss
# View Pledge [UI]

# What ❓
- Added the logic to show the proper `button string` and `button color` for `View your pledge, Manage, and Back this project.`

# How to QA? 🤔
- Go to Projects you've backed that have completed and when you click the project the reward button should read `View your pledge`

# Story 📖
[Move View my pledge into Rewards container](https://trello.com/c/Klxmcsgd/1280-move-view-my-pledge-into-rewards-container)

# See 👀
<img src=https://user-images.githubusercontent.com/16387538/59055613-3fccbc00-8864-11e9-8eda-c03e92f7887b.jpg width=300 />
